src/EnergyPlus/CMakeLists.txt

```diff
@@ -312,6 +312,10 @@ SET( SRC
   HighTempRadiantSystem.hh
   Humidifiers.cc
   Humidifiers.hh
+  HybridUnitaryAirConditioners.cc
+  HybridUnitaryAirConditioners.hh
+  HybridEvapCoolingModel.cc
+  HybridEvapCoolingModel.hh
   HybridModel.cc
```

Comment:
I don''t see an IDD object for a hybrid evap cooler?  Is this code active?  If not, it shouldn't added yet.